src/v/config/configuration.h

```diff
@@ -332,6 +332,8 @@ struct configuration final : public config_store {
     property<std::optional<size_t>>
       controller_log_accummulation_rps_capacity_configuration_operations;
 
+    bounded_property<uint32_t> crash_loop_max_limit;
```

Comment:
I can see why we'd want this as a cluster property for convenience, but if a node gets into a crash retry loop, they might intuitively expect to be able to get out of that state by disabling the property, which they can't do with a cluster config (unless they know how to reset in config_cache file, which I don't want to encourage).

So maybe we put it in node_config instead?  That's a relatively easy path for something like the k8s operator to drive, since it already writes out node configuration files.

Reply:
That makes sense. I added a mechanism that resets the crash loop detection on node configuration changes.